cmd/broker/main.go

```diff
@@ -10,14 +10,22 @@ import (
 	"github.com/docker/docker/client"
 	"github.com/pivotal-cf/brokerapi"
 
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
 	"github.com/cloudfoundry-incubator/blockhead/pkg/broker"
 	"github.com/cloudfoundry-incubator/blockhead/pkg/config"
+	"github.com/cloudfoundry-incubator/blockhead/pkg/containermanager"
 	"github.com/cloudfoundry-incubator/blockhead/pkg/containermanager/docker"
+	kcm "github.com/cloudfoundry-incubator/blockhead/pkg/containermanager/kubernetes"
 )
 
 func main() {
 	logger := lager.NewLogger("blockhead-broker")
 	logger.RegisterSink(lager.NewWriterSink(os.Stdout, lager.DEBUG))
+	logger.Info(os.Args[0])
+	logger.Info(os.Args[1])
+	logger.Info(os.Args[2])
```

Comment:
I can't figure out how to log the args so I can debug main failures in the tests.